bitbox-bridge/release/Makefile

```diff
@@ -17,7 +17,7 @@ release-linux:
 	docker run --user ${USER} --interactive --tty -v ${MOUNT} -w /bitbox-bridge ${CONTAINER} bitbox-bridge/release/linux/release.sh
 
 release-windows:
-	docker run --user ${USER} --interactive --tty -v ${MOUNT} -w /bitbox-bridge ${CONTAINER} bitbox-bridge/release/windows/release.sh
```

Comment:
Was this intentional?

Reply:
Ah, no - I locally removed the user as I am using podman. The other repos we have are compatible with either, but this one not yet. Reverted